docs/UserGuide.md

```diff
@@ -255,12 +255,12 @@ Adds a meeting to your meeting list.
 
 Format: `add t/TITLE from/FROM to/TO [d/DESCRIPTION] [c/CONTACTS] [l/LOCATION]`
 * `TITLE` is the main heading of the meeting.
-* `FROM` is the start date time of the meeting in DD-MM-YYYY HH:mm format.
-* `TO` is the end date time of the meeting in DD-MM-YYYY HH:mm format.
-* `FROM` and `TO` can be in the past but must not be earlier than the year 2019.
+* `FROM` is the start date time of the meeting in dd-MM-yyyy HH:mm format.
+* `FROM` can accept dates in the past but must not be earlier than the year 2019.
+* `TO` is the end time of the meeting in HH:mm format.
 * `DESCRIPTION` contains additional details about the meeting, e.g. agenda.
-* `CONTACTS` represents the contacts involved in the meeting. 
-* `CONTACTS` is a comma-separated string of names.
+* `CONTACTS` represents the contact(s) involved in the meeting. 
```

Comment:
Include optional plural for contacts in line 272

Reply:
Will amend to "`CONTACTS` has no relation to any contact in your contact list" to account for both

Reply:
Reworded to `As `CONTACTS` is not related to your contact list, you can include those not present in it.`